wodles/aws/aws_s3.py

```diff
@@ -69,6 +69,9 @@
 RETRY_CONFIGURATION_URL = 'https://documentation.wazuh.com/current/amazon/services/prerequisites/considerations.html#Connection-configuration-for-retries'
 DEPRECATED_MESSAGE = 'The {name} authentication parameter was deprecated in {release}. ' \
                      'Please use another authentication method instead. Check {url} for more information.'
+GUARDDUTY_URL = 'https://documentation.wazuh.com/current/amazon/services/supported-services/guardduty.html#amazon-configuration'
```

Comment:
Changed to have the base URL of the section. Since version `4.5`, the GuardDuty section will only describe the steps to configure the storage of GuardDuty findings in S3. Support for the older method via Kinesis will be provided but it won't be part of the documentation's new version.